src/Symfony/Bridge/Monolog/Handler/FingersCrossed/NotFoundActivationStrategy.php

```diff
@@ -20,23 +21,34 @@
  *
  * @author Jordi Boggiano <j.boggiano@seld.be>
  * @author Fabien Potencier <fabien@symfony.com>
+ * @author Pierrick Vignand <pierrick.vignand@gmail.com>
  */
-class NotFoundActivationStrategy extends ErrorLevelActivationStrategy
+/* final */ class NotFoundActivationStrategy extends ErrorLevelActivationStrategy implements ActivationStrategyInterface
```

Comment:
You should use a `@final` phpdoc instead.

Reply:
done